DuckDuckGo/TabViewController.swift

```diff
@@ -999,6 +1003,11 @@ extension TabViewController: WKNavigationDelegate {
     private func onWebpageDidFinishLoading() {
         os_log("webpageLoading finished", log: generalLog, type: .debug)
         
+        if let callback = onWebsiteLoaded {
+            callback()
+            onWebsiteLoaded = nil
+        }
```

Comment:
does this need to fall through to the stuff below?  maybe it should just return?